File: Exer10_1.cpp
#include <stdio.h>

int main ()
{	
	int i,vet[10];
	for (i = 0; i < 10; i++)
	{
		printf("Entre o valor de vet[%d]: ", i);
		scanf("%d", &vet[i]);
	}
	for (i = 9; i >= 0; i--)
		printf("%d\n", vet[i]);
	return(0);
}

File: Exer10_1.py
vet = []


def LerVetor(vetor):
    for i in range(10):
        vetor.append(int(input("Entre com o valor de vet[%d]:" % i)))


def MostrarVetor(vetor):
    for i in reversed(vetor):
        print(i)


LerVetor(vet)

MostrarVetor(vet)
